nck/writers/bigquery_writer.py

```diff
@@ -52,20 +47,11 @@ class BigQueryWriter(Writer, GoogleBaseClass):
     _client = None
 
     def __init__(
-        self,
-        dataset,
-        table,
-        bucket,
-        partition_column,
-        write_disposition,
-        location,
-        keep_files,
+        self, dataset, table, bucket, partition_column, write_disposition, location, keep_files,
     ):
 
         self._project_id = config.PROJECT_ID
```

Comment:
Here and in gcs_writer, there are some "config.PROJECT_ID" that we have to clean/refactor I think.

Reply:
Yes, indeed !
These variables are defined in the `nck/config.py` module:
```
# The below snippet is used in the following modules:
# - nck/readers/objectstorage_reader.py
# - nck/writers/gcs_writer.py
# - nck/writers/bigquery_writer.py

for key, var in os.environ.items():
    locals()[key] = var
```

But they are not defined explicitly, which is not optimal. Let me try to implement them directly as click options in the GCS and BQ writers.